tests/ignite/contrib/handlers/test_base_logger.py

```diff
@@ -58,34 +58,34 @@ def test_base_output_handler_setup_output_metrics():
 
     # Only metric_names
     handler = DummyOutputHandler("tag", metric_names=["a", "b"], output_transform=None)
-    metrics = handler._setup_output_metrics(engine=engine)
-    assert metrics == true_metrics
+    metrics = handler._setup_output_metrics(engine=engine, key_tuple=False)
+    assert metrics == {"tag/a": 0, "tag/b": 1}
```

Comment:
We can remove unused `true_metrics` definition, line 55

Reply:
Its being used in line 56 for passing in the metrics to the `State` right now. Should I remove it as a variable and directly pass the dictionary?

Reply:
Also, I added the type annotations to the new functions which was causing the check to fail.